<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>784</width>
         <height>538</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="scroolLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="ImageWidget" name="image" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDocument">
    <property name="title">
     <string>Document</string>
    </property>
    <addaction name="actionLocknext"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="separator"/>
    <addaction name="actionLock"/>
    <addaction name="actionSave"/>
    <addaction name="actionUnlock"/>
    <addaction name="actionUndochange"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionPageup"/>
    <addaction name="actionPagedown"/>
    <addaction name="separator"/>
    <addaction name="actionZoomin"/>
    <addaction name="actionZoomout"/>
    <addaction name="actionZoomfit"/>
    <addaction name="actionZoomorg"/>
    <addaction name="separator"/>
    <addaction name="actionRotateleft"/>
    <addaction name="actionRotateright"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDocks">
    <property name="title">
     <string>Docks</string>
    </property>
    <addaction name="actionAllowfloat"/>
    <addaction name="actionAllowmove"/>
    <addaction name="actionAllowclose"/>
    <addaction name="separator"/>
    <addaction name="actionLockdocks"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuDocument"/>
   <addaction name="menuView"/>
   <addaction name="menuDocks"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionZoomin">
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomout">
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionZoomfit">
   <property name="icon">
    <iconset theme="zoom-fit-best">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Fit to screen</string>
   </property>
  </action>
  <action name="actionZoomorg">
   <property name="icon">
    <iconset theme="zoom-original">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Original size</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Next document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset theme="go-previous">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Previous document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionLock">
   <property name="icon">
    <iconset theme="tools-check-spelling">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Lock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRotateleft">
   <property name="icon">
    <iconset theme="object-rotate-left">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Rotate left</string>
   </property>
  </action>
  <action name="actionRotateright">
   <property name="icon">
    <iconset theme="object-rotate-right">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Rotate right</string>
   </property>
  </action>
  <action name="actionUndochange">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Undo change</string>
   </property>
  </action>
  <action name="actionLocknext">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Lock next job</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="actionUnlock">
   <property name="icon">
    <iconset theme="edit-clear">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset theme="network-wired">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset theme="network-offline">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconncet from database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset theme="edit-find">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Import data</string>
   </property>
  </action>
  <action name="actionPageup">
   <property name="icon">
    <iconset theme="view-sort-ascending">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Next page</string>
   </property>
  </action>
  <action name="actionPagedown">
   <property name="icon">
    <iconset theme="view-sort-descending">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Previous page</string>
   </property>
  </action>
  <action name="actionAllowclose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Allow close</string>
   </property>
  </action>
  <action name="actionLockdocks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="changes-prevent">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Lock docks</string>
   </property>
  </action>
  <action name="actionAllowfloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-restore">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Allow float</string>
   </property>
  </action>
  <action name="actionAllowmove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-fullscreen">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Allow move</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>imagewidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
